fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -189,11 +200,33 @@ public Response head() {
 
             // we set the content-type explicitly to avoid content-negotiation from getting in the way
             builder.type(mediaType.toString());
+        } else {
+            final MediaType mediaType = RDFMediaType.TURTLE_TYPE;
+            final RdfStream stream = getResourceAsStream();
```

Comment:
I have created the following new ticket:
https://jira.duraspace.org/browse/FCREPO-2281

Let's:
* Close this PR
* Move the majority of the refactoring from here into a new PR against the above ticket
* Add a comment to https://jira.duraspace.org/browse/FCREPO-2248 reflecting the comments in this PR